### PR TITLE
Kamu 151 add set license event to dataset from UI

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/dataset-view/additional-components/overview-component/components/edit-license-modal/edit-license-modal.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(EditLicenseModalComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it("should create", () => {
+        expect(component).toBeTruthy();
+    });
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/overview-component/components/edit-license-modal/edit-license-modal.component.spec.ts:27]: Assuming we will have more tests, esp. validations

File: src/app/dataset-view/additional-components/overview-component/overview-component.ts [hunk 2]
```diff
@@ -99,4 +100,15 @@ export class OverviewComponent extends BaseComponent implements OnInit {
             modalRef.componentInstance as EditDetailsModalComponent
         ).datasetBasics = this.datasetBasics;
     }
+
+    public openLicenseModal() {
+        const modalRef: NgbModalRef = this.modalService.open(
+            EditLicenseModalComponent,
+        );
+        (modalRef.componentInstance as EditDetailsModalComponent).currentState =
```
Comment (zaychenko-sergei) [inline src/app/dataset-view/additional-components/overview-component/overview-component.ts:108]: This looks like another modal, not ours

File: src/app/common/app.values.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ export default class AppValues {
     public static readonly DEFAULT_USERNAME = "anonymous";
     public static readonly DEFAULT_AVATAR_URL =
         "https://avatars.githubusercontent.com/u/11951648?v=4";
+    public static readonly URL_REX_EXPRESSION =
```
Comment (zaychenko-sergei) [inline src/app/common/app.values.ts:10]: Weird name, please define a better one

**Review by zaychenko-sergei — CHANGES_REQUESTED** [review]

File: src/app/dataset-view/additional-components/overview-component/overview-component.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/overview-component/overview-component.ts:108]: Fixed

File: src/app/common/app.values.ts (hunk 3 above)
Comment (dmitriy-borzenko) [reply src/app/common/app.values.ts:10]: Fixed on URL_PATTERN

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**